Dockerfile

```
@@ -81,18 +81,18 @@ RUN conda install -y python=3.13 pip
-    make -j 4
-ENV PARSEC_EXECUTABLE=/app/parsec/build/parsec
-
 ENV CARGO_HOME="/app/.cargo"
 ENV RUSTUP_HOME="/app/.rustup"
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
 ENV PATH="/app/.cargo/bin:${PATH}"
 RUN rustup default 1.90.0
 RUN cargo install --locked kani-verifier && cargo kani setup
+
+COPY --chown=${USER_ID}:${GROUP_ID} parsec /app/parsec
+RUN cd /app/parsec && \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)